content/src/cloud/troubleshooting-cloud-backup/backup-troubleshooting.md

```diff
@@ -6,7 +6,7 @@ zendesk:
   labels: cloud
 ---
 
-**Finding your encryption key** - If you ever lose your encryption key, it can be re-downloaded in Home Assistant’s [Backups settings page](https://my.home-assistant.io/redirect/backup_config/). Navigate to the **Encryption key** section and either select to download your emergency kit or copy the key.
+**Finding your encryption key** - If you ever lose your encryption key, it can be re-downloaded in Home Assistant’s [**Backups settings**](https://my.home-assistant.io/redirect/backup_config/) page. Navigate to the **Encryption key** section and either select to download your emergency kit or copy the key.
```

Comment:
```suggestion
**Finding your encryption key** - If you ever lose your encryption key, it can be re-downloaded in Home Assistant’s [**Backups**](https://my.home-assistant.io/redirect/backup_config/) settings page. Navigate to the **Encryption key** section and either select to download your emergency kit or copy the key.
```

Reply:
nono, I'm referring to the actual UI string of this section here
![image](https://github.com/user-attachments/assets/c4730718-c815-463f-859d-96a0d82dda99)

Reply:
but yeah, even then, there is a typo :laughing: